services/wallet/aws/client.go

```diff
@@ -0,0 +1,29 @@
+package aws
```

Comment:
https://github.com/brave-intl/bat-go/pull/1569

This is looking very similar to the implementation here for getting the geo information for custodial linkings.  As the ticket mentions tho, these two lists should be different.  I suppose maybe we could consolidate implementations?

Reply:
yes, I was going to refactor out the params one so the use the same aws client